src/main.rs

```diff
@@ -1,7 +1,16 @@
 use cli_batteries::version;
 use kzg_ceremony_sequencer::async_main;
+use opentelemetry::{global, sdk::export::trace::stdout::PipelineBuilder};
 
 #[allow(dead_code)] // Entry point
 fn main() {
+    // Install OpenTelemetry tracer to enable trace ids
+    PipelineBuilder::default()
+        .with_writer(std::io::sink())
+        .install_simple();
```

Comment:
`cli_batteries` should already take care of setup/teardown of the OTLP pipeline: https://github.com/recmo/cli-batteries/blob/main/src/trace/open_telemetry.rs

Just need to enable the `otlp` feature flag on it.